### PR TITLE
HVG-622: Adyen Payout, Connected/Pending

### PR DESCRIPTION
**Pull request by hedvigoscar**
### Checklist

- [X] Functionality is covered by an integration test
- [X] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt [hunk 1]
```diff
@@ -361,38 +365,58 @@ class PaymentAdapter(
...
+                if (data !is PaymentModel.PayoutConnectionStatus) {
                     return invalid(data)
                 }
 
-                root.setText(
-                    when (data) {
-                        PaymentModel.PayoutDetailsParagraph.Add -> R.string.payment_screen_pay_out_footer_not_connected
+                when (data.status) {
+                    PayoutMethodStatus.ACTIVE -> {
+                        root.setText(R.string.payment_screen_pay_connected_label)
```
Comment (hugokallstrom) [inline app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt:387]: Remove new line

File: app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt [hunk 2]
```diff
@@ -361,38 +365,58 @@ class PaymentAdapter(
...
+                        root.putCompoundDrawablesRelativeWithIntrinsicBounds(
+                            start = R.drawable.ic_checkmark_in_circle
+                        )
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt:390]:
```suggestion
                        root.putCompoundDrawablesRelativeWithIntrinsicBounds(start = R.drawable.ic_checkmark_in_circle)
```

As such? 🙂

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt:390]: @hugokallstrom

Comment (hugokallstrom) [reply app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt:390]: Sorry I meant between `root.setText(R.string.payment_screen_pay_connected_label)` and 
```
root.putCompoundDrawablesRelativeWithIntrinsicBounds(
                            start = R.drawable.ic_checkmark_in_circle
                        )
```

File: app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/profile/ui/payment/PaymentAdapter.kt:387]: I don't think there's actually a real newline between here 😬

**Pull request merged**